### ISSUE TITLE
How use dropout in train model?

### ISSUE THREAD

**Issue by BENZJ** [tensorflow/cleverhans#368]
I have read the  tutorials of MNIST, but I can't find how I can use dropout in my train model, because my model have used dropout function. Could you give me some example. Thank you

**Comment by goodfeli** [comment]
What do you need to know specifically?
